src/tags_int.cpp

```diff
@@ -2585,12 +2585,22 @@ namespace Exiv2 {
     URational exposureTime(float shutterSpeedValue)
     {
         URational ur(1, 1);
-        double tmp = std::exp(std::log(2.0) * static_cast<double>(shutterSpeedValue));
+        const double tmp = std::exp(std::log(2.0) * static_cast<double>(shutterSpeedValue));
         if (tmp > 1) {
-            ur.second = static_cast<long>(tmp + 0.5);
+            // Add 0.5 for rounding.
+            const double x = tmp + 0.5;
+            // Check that x is within the range of a uint32_t before casting.
+            if (0 <= x && x <= std::numeric_limits<uint32_t>::max()) {
```

Comment:
```suggestion
            if (x <= std::numeric_limits<uint32_t>::max()) {
```
This is to fix the LGTM warning because `x>=0` is always true in the `tmp > 1` branch